src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetChangelogPage.tsx

```diff
@@ -74,49 +78,55 @@ export default function ReleaseApiDataSetChangelogPage() {
       >
         Back to API data set details
       </Link>
+
       <LoadingSpinner loading={isLoadingDataSet || isLoadingChanges}>
-        <div className="govuk-grid-row">
-          <div className="govuk-grid-column-three-quarters">
-            <span className="govuk-caption-l">API data set changelog</span>
-            <h2>{dataSet?.title}</h2>
-          </div>
-        </div>
+        {!errorFetchingDataSet && !errorFetchingChanges ? (
+          <>
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-three-quarters">
+                <span className="govuk-caption-l">API data set changelog</span>
+                <h2>{dataSet?.title}</h2>
```

Comment:
A benefit of checking that `dataSet` is defined (after checking loading has finished) is that the TS compiler can also treat the variable as defined and we don't need to use optional chaining here

Reply:
I've also added `dataSetVersion` as part of the check, as this must be defined if `dataSet is defined, unlesss something has gone disastrously wrong!  We can now remove a bunch of null-checking for dataSetVersion as well.